src/screens/map-screen/map-screen.component.tsx

```diff
@@ -49,6 +53,14 @@ const MapScreen = () => {
       floors: []
     }
   );
+  const [destination, setDestination] = useState<POI>(null);
+  const [initialLocation, setInitialLocation] = useState<POI | UserLocation>(
+    null
+  );
+  const [markerSetsDestination, setMarkerSetsDestination] = useState<Boolean>(
```

Comment:
what does markerSetsDestination do exactly? consider leaving a comment.

Reply:
this is an example of bad naming that should be changed